.eslintignore

```diff
@@ -1,2 +1,2 @@
-node_modules
```

Comment:
why did you remove these two entries?

Reply:
Ignored by default.
https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file